spec/_attachments/ic.did

```diff
@@ -241,5 +258,8 @@ service ic : {
     sender_canister_version : opt nat64;
   }) -> (record {canister_id : canister_id});
   provisional_top_up_canister :
-    (record { canister_id: canister_id; amount: nat }) -> ();
+    (record { canister_id : canister_id; amount: nat }) -> ();
+
+  // canister logging
+  fetch_logs : (record { canister_id : canister_id }) -> (fetch_logs_response) query;
```

Comment:
```suggestion
  fetch_canister_logs : (record { canister_id : canister_id }) -> (fetch_logs_response) query;
```

Would it make sense to update the method name to make it clear that it's fetching the logs of a canister? Maybe in the future we want to add a way to fetch logs for the replicas, for instance.

Reply:
Done.